.github/workflows/conformance-aks-v1.10.yaml

```
@@ -142,7 +142,8 @@ jobs:
             --azure-resource-group ${{ env.name }} \
             --wait=false \
             --rollback=false \
-            --config monitor-aggregation=none"
+            --config monitor-aggregation=none" \
+            --base_version=v1.10.5
```

```suggestion
            --base-version=v1.10.5
```

same for the other workflows

---

:facepalm: good eye